packages/common/src/eips/1559.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "EIP-1559",
+    "number": 1559,
+    "comment": "Fee market change for ETH 1.0 chain",
+    "url": "https://eips.ethereum.org/EIPS/eip-1559",
+    "status": "Review",
+    "minimumHardfork": "london",
+    "gasConfig": {
+      "baseFeeMaxChangeDenominator": {
+        "v": 8,
+        "d": "Maximum base fee change denominator"
+      },
+      "elasticityMultiplier": {
```

Comment:
I would name this `gasTargetElasticityMultiplier`, otherwise it lacks semantics if just read on its own (just learning from the experience with other parameter names from common - some inherited from former devs - where it is just not possible to get the context when reading).

Reply:
On a second thought having this in the `gasConfig` section maybe that's enough context. Will leave as-is.